pyk/src/pyk/__main__.py

```diff
@@ -389,6 +393,60 @@ def exec_json_to_kore(options: JsonToKoreOptions) -> None:
     sys.stdout.write('\n')
 
 
+def exec_parse_outer(options: ParseOuterOptions) -> None:
+    main_file_dir = Path(options.main_file.name).resolve().parent
+    md_selector = options.md_selector
+    required_files = []
+    search_paths = [main_file_dir]
+    for include in options.includes:
+        include_path = Path(include)
+        try:
+            check_dir_path(include_path)
+        except ValueError:
+            _LOGGER.critical(f"Could not find directory '{include}' passed to -I")
+            exit(1)
```

Comment:
Worth noting this isn't what `-I` does in the existing compiler frontend, nor in GCC / clang. Could maybe be a lower level warning rather than an error eventually?

Reply:
Yep, I can just make this a warning.